index.js

```diff
@@ -88,6 +247,43 @@ class instance extends instance_skel {
 					}
 				]
 			},
+			'setTimecode': {
+				label: 'Set Timecode Value',
+				options: [
+					{
+						type: 'dropdown',
```

Comment:
Is a dropdown necessary here?  We have `this.REGEX_TIMECODE` available to do a text input.

Reply:
ohhh ... nevermind, I think.  Misread how this parameter is defined.

Reply:
This was done that way because the KiPro only allows you to set the starting hour and it then sets the minutes, seconds, and frames to 00. I did it to make it more clear to the end user what the timecode would be set to.